frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts

```diff
@@ -150,16 +150,36 @@ const osUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) => {
   if (!hasVMSettingsValueChanged(prevState, state, id, VMSettingsField.OPERATING_SYSTEM)) {
     return;
   }
-  const os = iGetVmSettingValue(state, id, VMSettingsField.OPERATING_SYSTEM);
-  const isWindows = os && os.startsWith('win');
+  const os: string = iGetVmSettingValue(state, id, VMSettingsField.OPERATING_SYSTEM);
+  const isWindows: boolean = os && os.startsWith('win');
```

Comment:
```suggestion
  const isWindows: boolean = !!os?.startsWith('win');
```

I meant this so we are not saving undefined/null there.

one side note about VM Wizard store: `undefined` updates are not acknowledged but all other values are (including `null` and `''`)

Reply:
:+1: